src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;

```

Comment:
Consider using a constant for the value of PI, as magic numbers should be avoided. While Math.PI is a well-known constant, defining it as a static final variable within the class can improve readability and maintainability, especially if you need to change the precision or the value in the future.